repo-scripts/size-analysis/package.json

```diff
@@ -27,7 +27,7 @@
     "rollup-plugin-typescript2": "0.29.0",
     "@rollup/plugin-virtual": "2.0.3",
     "webpack": "4.44.2",
-    "@types/webpack": "4.41.25",
+    "@types/webpack": "5.28.0",
```

Comment:
Is it required to pass CI? This is a wrong change because we are not using webpack@5. I will be okay with it if it's required to pass CI for some unknown reason. I don't want us to waste time on it and we should be updating to `webpack@5` soon.

Reply:
I am not sure if 5.x is required, but the version that was installed without the yarn.lock file caused compilation errors. I can try an older 4.x version, but we should be safe since the code itself did not change.

Reply:
The problem is here (from the original yarn.lock):

```
"@types/webpack@4.41.25":
  version "4.41.25"
  resolved "https://registry.npmjs.org/@types/webpack/-/webpack-4.41.25.tgz#4d3b5aecc4e44117b376280fbfd2dc36697968c4"
  integrity sha512-cr6kZ+4m9lp86ytQc1jPOJXgINQyz3kLLunZ57jznW+WIAL0JqZbGubQk4GlD42MuQL5JGOABrxdpqqWeovlVQ==
  dependencies:
    "@types/anymatch" "*"
    "@types/node" "*"
    "@types/tapable" "*"
    "@types/uglify-js" "*"
    "@types/webpack-sources" "*"
    source-map "^0.6.0"
```

Webpack 4.x does not compile against tapable 2.x. The old version was 1.06 (before removing the lock filed).

These are some of the errors:

```
../../node_modules/@types/webpack/index.d.ts:1106:32 - error TS2314: Generic type 'HookMap<H>' requires 1 type argument(s).

1106                 callAnyMember: HookMap;
                                    ~~~~~~~
```